product_landing_page/index.css

```diff
@@ -0,0 +1,55 @@
+@import "https://fonts.googleapis.com/css?family=Lato:400,700";
+
+body {
+  background-color: #eee;
+  margin: 0;
+  box-sizing: border-box;
+  font-family: "Lato", sans-serif;
+}
+
+.nav {
+  display: flex;
+  background-color: #ccc;
+  box-shadow: 0px 1px #888888;
+  position: sticky;
+  top: 0;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  width: 90%;
```

Comment:
Tutaj trzeba dać `width: 100%;` i `padding: 0`. Podejrzewam, że odstęp itema "Pricing" od prawej krawędzi ekranu chciałeś ustawić poprzez to `width: 90%`, ale to bardziej margin, więc tutaj będzie potrzebna korekta tego odstępu. Do tego powodowało to problemu przy RWD.

Przed:
![przed](https://user-images.githubusercontent.com/23278550/50293460-ec141500-0473-11e9-948c-342ec845bcf1.png)
 
Po: 
![po](https://user-images.githubusercontent.com/23278550/50293472-f3d3b980-0473-11e9-93bf-7169842d565c.png)